File: app/src/main/java/com/expansion/lg/kimaru/expansion/mzigos/ChewReferral.java
package com.expansion.lg.kimaru.expansion.mzigos;

/**
 * Created by kimaru on 5/12/17.
 */

public class ChewReferral {
    String id, name, phone, title, country, recruitmentId;
    Integer synced;
    int color = -1;

    public ChewReferral(){}

    public ChewReferral(String id, String name, String phone, String title, String country,
                        String recruitmentId, Integer synced){
        this.id = id;
        this.name = name;
        this.phone = phone;
        this.title = title;
        this.country = country;
        this.recruitmentId = recruitmentId;
        this.synced = synced;
    }

    public Integer getSynced() {
        return synced;
    }

    public void setSynced(Integer synced) {
        this.synced = synced;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getRecruitmentId() {
        return recruitmentId;
    }

    public void setRecruitmentId(String recruitmentId) {
        this.recruitmentId = recruitmentId;
    }

    public int getColor() {
        return color;
    }

    public void setColor(int color) {
        this.color = color;
    }
}

File: app/src/main/java/com/expansion/lg/kimaru/expansion/mzigos/ChewReferral.kt
package com.expansion.lg.kimaru.expansion.mzigos

/**
 * Created by kimaru on 5/12/17.
 */

class ChewReferral {
    lateinit var id: String
    lateinit var name: String
    lateinit var phone: String
    lateinit var title: String
    lateinit var country: String
    lateinit var recruitmentId: String
    //Some mapping Details that we shall use, esp for UG
    lateinit var county: String
    lateinit var district: String
    lateinit var subCounty: String
    lateinit var communityUnit: String
    lateinit var village: String
    lateinit var mapping: String
    lateinit  var lat: String
    lateinit  var lon: String
    lateinit  var mobilization: String
    var synced: Int? = null
    var color = -1

    constructor() {}

    constructor(id: String, name: String, phone: String, title: String, country: String,
                recruitmentId: String, synced: Int?, county: String, district: String,
                subCounty: String, communityUnit: String, village: String, mapping: String,
                lat: String, lon: String, mobilization: String) {
        this.id = id
        this.name = name
        this.phone = phone
        this.title = title
        this.country = country
        this.recruitmentId = recruitmentId
        this.synced = synced
        this.county = county
        this.district = district
        this.subCounty = subCounty
        this.communityUnit = communityUnit
        this.village = village
        this.mapping = mapping
        this.lat = lat
        this.lon = lon
        this.mobilization = mobilization
    }
}
